doc-src/ch-pull2dir_desc.rst

```diff
@@ -0,0 +1,59 @@
+Synopsis
+========
+
+::
+
+  $ ch-pull2dir IMAGE[:TAG] DIR
+
+Description
+===========
+
+Pulls a Docker image named :code:`IMAGE[:TAG]` from Docker Hub, flattens it
+into a Charliecloud tarball and extracts the tarball into a subdirectory of 
+:code:`DIR`.
+
+Inside :code:`DIR`, the subdirectory will be created whose name corresponds to
+the name of the tarball with the :code:`.tar.gz` suffix removed. If such a
+directory exists already and appears to be a Charliecloud container image, it
+is removed and replaced. If the existing directory doesn't appear to be a
+container image, the script aborts with an error.
+
+Sudo privileges are required to run the :code:`docker pull` command.
```

Comment:
There's a fair amount of copy and paste description here. I would refer people to the underlying command documentation instead.

For example, the man page for `ch-build2dir` is a bit stale, but it's still a decent model.